src/Type/AuxiliaryDocument.php

```diff
@@ -4,15 +4,15 @@
 
 namespace OpenEuropa\EPoetry\Type;
 
-class AuxiliaryDocument extends DgtDocument
+class AuxiliaryDocument extends \OpenEuropa\EPoetry\Type\DgtDocument
```

Comment:
Isn't that a regression having the full namespace `\OpenEuropa\EPoetry\Type\DgtDocument`?

Reply:
As long as this is not fixed in php-cs-fixer, we shouldn't modify them...